core/iwasm/compilation/aot_compiler.c

```diff
@@ -785,19 +945,31 @@ aot_compile_func(AOTCompContext *comp_ctx, uint32 func_index)
                 break;
 
             case WASM_OP_BR:
+            {
+                frame_ip_org = frame_ip - 1;
```

Comment:
Why we need to `-1` here but in some other opcode (WASM_OP_REF_FUNC) don't `-1`?

Reply:
Here I just want to commit the beginning of the bytecode to stack frame. There are errors in handling REF_FUNC/CALL_REF, I will fix them soon.

Reply:
Got it. Then the GC related opcode seems need to -2 since their opcode are 2 bytes

Reply:
Yes, `frame_ip_org = frame_ip - 1` was added after reading the GC PREFIX and before reading each sub opcode of GC, see L1319.